### PR TITLE
Use Static Requires

### PR DESCRIPTION
**Pull request by Noldaru**
Modified requires for 'types' modules to static as dynamic requires are not supported with Browserify.

### CODE REVIEW

File: lib/types/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    dds: require('./dds'),
+    gif: require('./gif'),
+    ico: require('./ico'),
+    jpg: require('./jpg'),
+    png: require('./png'),
+    psd: require('./psd'),
+    svg: require('./svg'),
+    tiff: require('./tiff'),
+    webp: require('./webp')
+};
```
Comment (LinusU) [inline lib/types/index.js]: missing newline at end of file, and the other files are using two spaces as indentation

**Comment by danielweck** [comment]
Excellent, I hope this gets merged soon so I can kill my temporary-fix fork :)
https://github.com/image-size/image-size/issues/39#issuecomment-351431024

File: lib/detector.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
 'use strict';
 
 var typeMap = {};
-var types = require('./types');
+var handlers = require('./types');
+var types = Object.keys(typeMap);
```
Comment (eiwi1101) [inline lib/detector.js]: Did you mean for this to be:

```javascript
var types = Object.keys(handlers);
```

This might be what's holding up the specs.

**Comment by netroy** [comment]
fixed in #105 & released as `0.6.2`

**Pull request closed without merging**